jest.config.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  preset: 'ts-jest',
+  preset: 'ts-jest/presets/js-with-ts',
```

Comment:
why is this necessary?

Reply:
this transpilesjs files, including polkadot dependencies, to CommonJS syntax. This is because jest can't really handle esm yet